shim/tests/src/test_handle_system_kvm_get_msr_index_list.cpp

```diff
@@ -43,19 +50,142 @@ namespace shim
     [[nodiscard]] constexpr auto
     tests() noexcept -> bsl::exit_code
     {
-        bsl::ut_scenario{"description"} = []() noexcept {
+        init_tests();
+
+        bsl::ut_scenario{"success"} = []() noexcept {
+            bsl::ut_given{} = [&]() noexcept {
+                kvm_msr_list mut_args{};
+                bsl::ut_when{} = [&]() noexcept {
+                    mut_args.indices = g_mut_msr_indices.front_if();
+                    mut_args.nmsrs = INIT_NMSRS.get();
+                    g_mut_val = bsl::safe_u64::magic_2().get();
+                    bsl::ut_then{} = [&]() noexcept {
+                        bsl::ut_check(
+                            SHIM_SUCCESS == handle_system_kvm_get_msr_index_list(&mut_args));
```

Comment:
Please use the pattern in the other unit tests where the handle function is called using handle(), which is function pointer to the actual handle function that you want to use. It removes the extra new lines. 

https://github.com/Bareflank/MicroV/blob/master/shim/tests/src/test_handle_vcpu_kvm_get_regs.cpp#L52